lib/common/ffi_subscriber.dart

```diff
@@ -3,12 +3,6 @@ import 'package:web_socket_channel/status.dart' as status;
 import 'common.dart';
 import 'common_desktop.dart';
 
-extension BoolParsing on String {
-  bool parseBool() {
-    return this.toLowerCase() == 'true';
-  }
-}
-
 class FfiValueNotifier<T> extends SubscribedNotifier<T?> {
   FfiValueNotifier(
```

Comment:
I think we can remove most of the arguments now, Since we are using only `defaultValue` and `removeFromCache`

Reply:
Done